pkgs/by-name/ap/apotris/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  meson,
+  ninja,
+  python3,
+  cmake,
+  pkg-config,
+  xxd,
+  libopus,
+  libogg,
+  zlib,
+  SDL2,
+  SDL2_mixer,
+  libopenmpt,
+  libXrandr,
+  libXext,
+  libXfixes,
+  libXcursor,
+  libXi,
+  libXScrnSaver,
+  makeDesktopItem,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apotris";
+  version = "4.0.2";
+
+  src = fetchFromGitea {
+    domain = "gitea.com";
+    owner = "akouzoukos";
+    repo = "apotris";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-w9uv7A1UIn82ORyyvT8dxWHUK9chVfJ191bnI53sANU=";
+    fetchSubmodules = true;
+  };
+
+  postPatch = ''
+    patchShebangs tools/bin2s.py
+  '';
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    python3
+    cmake
```

Comment:
Why cmake?

Reply:
> Why cmake?

without cmake:
```
apotris> Program xxd found: YES (/nix/store/kbm4h3cbywk2s8mgsmamqb2n5c571bz7-xxd-tinyxxd-1.3.5/bin/xxd)
apotris> meson.build:128: WARNING: Consider using the built-in warning_level option instead of using "-Wall".
apotris> Found pkg-config: YES (/nix/store/02vzklfgfpaf457rq927fq3jj5hn133f-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
apotris> WARNING: Static library 'opus' not found for dependency 'opus', may not be statically linked
apotris> Run-time dependency opus found: YES 1.5.2
apotris> WARNING: Static library 'ogg' not found for dependency 'ogg', may not be statically linked
apotris> Run-time dependency ogg found: YES 1.3.5
apotris> Did not find CMake 'cmake'
apotris> Found CMake: NO
```
and build failed.